src/styles/blocks/header.scss

```diff
@@ -0,0 +1,54 @@
+.header__nav {
+  position: relative;
+}
+
+.header {
+  display: flex;
+  // align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+}
+
+.nav__list {
+  text-decoration: none;
+  margin-left: 20px;
+  text-transform: uppercase;
+  font-size: 12px;
+  font-weight: 500;
+  color: #000;
+  padding: 23px 0;
+  line-height: 60px;
+}
+
+.nav__list:hover {
+  color: rgba(0, 172, 220, 1);
+}
+
+.nav__ul {
+  display: flex;
+  list-style-type: none;
+  justify-content: space-between;
+  padding: 0;
+}
```

Comment:
create a separate file for the nav block and make everything according to BEM.

Reply:
+